lms/djangoapps/courseware/views.py

```diff
@@ -395,6 +395,9 @@ def _index_bulk_op(request, course_key, chapter, section, position):
             'reverifications': fetch_reverify_banner_info(request, course_key),
         }
 
+        # Check for course required content (such as an Entrance Exam)
+        chapter, section = override_with_required_content(course_module, course, request.user, chapter, section)
```

Comment:
I think we'll want a FEATURE flag check here to make sure the entrance exam is turned on. Otherwise this courseware inspection is a little bit expensive with Mongo database roundtrips